modules/ocf_ceph/manifests/init.pp

```diff
@@ -0,0 +1,26 @@
+# Base class for ceph
+
+class ocf_ceph {
+  require ocf::packages::ceph
+
+  $ceph_fsid = lookup('ceph::fsid')
+
+  $ceph_mons_list = lookup('ceph::mons')
+  $ceph_mons = $ceph_mons_list.join(',')
+  $ceph_mon_ips = $ceph_mons_list.map
+    |$node| { ldap_attr($node, 'ipHostNumber') }.join(',')
```

Comment:
nit: weird formatting. we usually would put `|$node|` on the lame line as the map function

Reply:
Agreed, that's how ruby does it normally (which puppet is based off)

Reply:
Personally I'd go with this:

```puppet
$ceph_mon_ips = $ceph_mons_list.map |$node| {
  ldap_attr($node, 'ipHostNumber')
}.join(',')
```